Sources/GRPCInProcessTransport/InProcessServerTransport.swift

```diff
@@ -55,12 +62,9 @@ public struct InProcessServerTransport: ServerTransport, Sendable {
     }
   }
 
-  /// Return a new ``RPCAsyncSequence`` that will contain all published ``RPCStream``s published
-  /// to this transport using the ``acceptStream(_:)`` method.
-  ///
-  /// - Returns: An ``RPCAsyncSequence`` of all published ``RPCStream``s.
-  public func listen() async throws -> RPCAsyncSequence<RPCStream<Inbound, Outbound>> {
-    RPCAsyncSequence(wrapping: self.newStreams)
+  // Signal that the stream is up and running.
+  public func listen() async {
+    self.eventStreamContinuation.yield(.success(RPCAsyncSequence(wrapping: self.newStreams)))
```

Comment:
This transport doesn't really have any long-running work to do here. Is it really necessary to not return until `stopListening()` has been called? The requests will be handled anyways because the server will still be holding the accepted streams sequence.

Reply:
It doesn't here, but in general they will. Maybe the distinction doesn't matter so much, need to think about it some more.